docs/frontend-api/introduction-to-frontend-api.md

```diff
@@ -1,21 +1,148 @@
 # Introduction to Frontend API
 
-Shopsys Framework Frontend API is an interface to the application that is dedicated for integration with your third party systems systems (like IS, storage system, ERP, ...).
+Shopsys Framework Frontend API is an interface to the application that is used for integration with external shop frontend, for example, JS storefront od mobile app.
 We use [GraphQL](https://graphql.org) (implemented using [overblog/GraphQLBundle](https://github.com/overblog/GraphQLBundle)).
 
+GraphQL is a query language for APIs and provides an understandable description of the data in the API,
+which gives clients the power to ask for exactly what they need and nothing more,
+makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+You should see the [GraphQL documentation](https://graphql.org/learn/) for more information and learn how to query the data you need.
+
 ## Installation
-Frontend API package is installed by default in `shopsys/project-base`.
-In case you want to add the package into already existing project, you should follow [upgrade instructions](https://github.com/shopsys/shopsys/blob/9.0/upgrade/UPGRADE-v9.0.0-dev.md)
+Frontend API package is installed by default in `shopsys/project-base`.  
+In case you want to add the package into an already existing project, you should follow [upgrade instructions](https://github.com/shopsys/shopsys/blob/9.0/upgrade/UPGRADE-v9.0.0-dev.md)
 
 ## Configuration
+<!--- TODO add after configuration refactoring -->
```

Comment:
TODO here

Reply:
Content added in ee698a5e1521a7dea930e907e14869dd9fa67a56